### PR TITLE
Add support for Knights Mill

### PR DESCRIPTION
**Pull request by michalbiesek**
### Types of changes

- [ ] Bugfix (non-breaking change which fixes issue linked in Description above)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance)
- [ ] Other

### Checklist

- [ ] Code compiles without errors
- [ ] All tests pass locally with my changes (see TESTING section in CONTRIBUTING file)
- [ ] Created tests which will fail without the change (if possible)
- [ ] Extended the README/documentation (if necessary)
- [ ] All newly added files have proprietary license (if necessary)
- [ ] All newly added files are referenced in MANIFEST files (if necessary)

## Further comments

---
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/434)

### CODE REVIEW

**Review by bratpiorka — COMMENTED** [review]
:lgtm:

> Reviewed 10 of 10 files at r1.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/434#-:-MOjhDR_77LjYGfbaBbh:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @PatKamin)

File: utils/qemu/topology/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# README
+
+This is utils/qemu/topology/README.
+
+XML files in this directory simulates different memory architectures.
+
+- knm_all2all.xml - Knights Mill, cluster mode: All2All
+- knm_snc2.xml - Knights Mill, cluster mode: SNC-2
+- knm_snc4.xml - Knights Mill, cluster mode: SNC-4
```
Comment (PatKamin) [inline utils/qemu/topology/README.md:9]: Add an information that these architectures are for hbw setup.

**Review by michalbiesek — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/434)* status: 9 of 10 files reviewed, 1 unresolved discussion (waiting on @bratpiorka and @PatKamin)

___
*[utils/qemu/topology/README.md, line 9 at r1](https://reviewable.io/reviews/memkind/memkind/434#-MOjgOXR9obJRde3MICB-r1-9:-MOjyK362wgXyf-8Ncrc:b-xtk3ey) ([raw file](https://github.com/memkind/memkind/blob/99e23ba077a860aece9fdb0de48188fb4a6f31be/utils/qemu/topology/README.md#L9)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

Add an information that these architectures are for hbw setup.
</blockquote></details>

I added information about supported memory,

**Review by PatKamin — COMMENTED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/434#-:-MOjzRHN6Wx_m1QUiwz-:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @michalbiesek)

**Pull request merged**